cmd/kubeops/main.go

```diff
@@ -76,6 +77,18 @@ func main() {
 	addRoute("PUT", "/namespaces/{namespace}/releases/{releaseName}", handler.OperateRelease)
 	addRoute("DELETE", "/namespaces/{namespace}/releases/{releaseName}", handler.DeleteRelease)
 
+	// Backend routes unrelated to kubeops functionality.
+	// TODO(mnelson): Once the helm3 support is complete and tiller-proxy is being removed,
```

Comment:
This TODO can perhaps be removed?

Reply:
I agree, I just didn't know if you were thinking on something else